src/app/components/navigation/navigation.component.html

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-light bg-light fixed-top navbar-expand-lg" [ngClass]="{'shrink': isShrunk}" #navView>
+<nav class="navbar fixed-top navbar-expand-lg" [ngClass]="{'shrink': isShrunk}" #navView>
   <a href="https://github.com/zonemaster/zonemaster/" class="github-corner d-none d-lg-block d-xl-block" aria-label="View source on Github">
```

Comment:
The Github thing should have inverted colors in dark theme (Octocat on black with a white background)

Reply:
(Ignore comments if https://github.com/zonemaster/zonemaster-gui/pull/383 is merged)